config/locales/doorkeeper.en.yml

```diff
@@ -0,0 +1,123 @@
+en:
```

Comment:
Good spotting Rich!

It's weird to me that it's in here at all. I guess it's here so you can easily modify them.

 If we delete it, I wonder if doorkeeper will fallback to strings within it's own gem? They would have this translated already.